src/GraphQL/Execution/ExecutionHelper.cs

```diff
@@ -382,6 +408,9 @@ public static object CoerceValue(ISchema schema, IGraphType type, IValue input,
             return fields;
         }
 
+        /// <summary>
+        /// TODO
```

Comment:
see http://spec.graphql.org/June2018/#sec-Field-Collection and http://spec.graphql.org/June2018/#CollectFields()

Reply:
```suggestion
        /// See http://spec.graphql.org/June2018/#sec-Field-Collection and http://spec.graphql.org/June2018/#CollectFields()
```
Is this what you suggest at this time?

Reply:
I think it is OK to just point to the algorithm from the spec.

Reply:
Sure you can't come up with some type of description?

Reply:
You can copy-paste description from the spec :)